pythonforandroid/recipes/python3/__init__.py

```diff
@@ -6,14 +6,29 @@
 
 class Python3Recipe(GuestPythonRecipe):
     '''
-    The python3's recipe.
+    The python3's recipe
+    ^^^^^^^^^^^^^^^^^^^^
 
-    .. note:: This recipe can be built only against API 21+. Also, in order to
-        build certain python modules, we need to add some extra recipes to our
-        build requirements:
+    The python 3 recipe can be built with some extra python modules, but do so,
+    we need some libraries. Per default, we ship the python3 recipe with some
+    common libraries, defined at ``depends``.We also support some optional
```

Comment:
I would say `defined in` but I suck at grammar. Also missing a space between the dot and "We"

Reply:
I'm not sure about this but what I'm sure is that your English is a lot better than mine...so let's go for `in`, thanks!!! :smile: